jest.config.js

```diff
@@ -8,9 +8,9 @@ module.exports = {
 
   moduleFileExtensions: ['js', 'json', 'jsx'],
 
-  setupFiles: ['<rootDir>/test/enzyme.config.js'],
+  setupFiles: ['<rootDir>/tests/enzyme.config.js'],
 
-  testEnvironment: 'jsdom',
+  testEnvironment: 'node',
```

Comment:
How come you changed the testEnvironment from jsdom to node?

Reply:
So that jest can handle testing mongoose